cirq-core/cirq/transformers/gauge_compiling/gauge_compiling.py

```diff
@@ -201,6 +209,102 @@ def __call__(
                 new_moments.extend(_build_moments(right))
         return circuits.Circuit.from_moments(*new_moments)
 
+    def randomized_compiling(
```

Comment:
I think a better name would be `as_sweep`

Reply:
Good point, done.